build.sbt

```diff
@@ -36,7 +36,7 @@ lazy val root = (project in file("."))
     TestAssets / JsEngineKeys.npmNodeModules := Nil
   )
 
-val tortoiseVersion = "1.0-88d5c3c"
+val tortoiseVersion = "1.0-2330ccf"
```

Comment:
This change shouldn't be necessary now, the latest Tortoise that the `master` branch is on has the latest I18N bundles.